guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+# Getting Started with GitHub
+
+What is GitHub, and how do you use it to interact with and contribute to SourceCred?
+This guide will give you an introduction to GitHub and SourceCred.
+
+## Why do we use GitHub?
+
+For now, think of GitHub as a place to collaborate. 
+If you had a bunch of documents that you wanted to collaborate on, what would you do?
+You might consider putting them in a shared file service like [Google Docs](https://docs.google.com/), or perhaps keeping them as Word documents on your local machine. 
+Although these platforms offer general collaborative editing capabilities, GitHub
+offers a bit more:
+
+### Permissionless collaboration
+
+Anyone can propose changes. You don't need to be given permission to view or edit
+a document, as is required by some of the platforms mentioned previously.
+
+### More advanced reviewing tools
+
+When you open up a Google Doc, you might be able to comment or write in revision mode, but
+when you do so, you are working with the entire document. GitHub allows
+review on the level of change. Along with this more detailed ability to edit,
+there is an integrated approval workflow, straightforward visualizations of what changed 
+(these are called "diffs"), and powerful automatic testing (called continuous integration or CI).
+
+### More detailed history
+
+You are going to have basic questions like:
+
+ - Who added this piece?
+ - When and why was this added?
+
+And GitHub has built-in tools to do this (one such action is called [git blame](https://help.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file).)
+
+### Easy for many people to collaborate at once
+
+You can work in your own "alternate reality" without interfering with other work, even simultaneously. When ready these alternatives can be merged back together.
+You can also work on changes out of order, although this is more of an advanced topic.
+
+### Web Interfaces
+
+By way of some helpful features, the text files that you are working on can be served as web pages
+directly from where they live. You can read about this feature, called [GitHub Pages](https://pages.github.com/).
+
+To summarize and answer this question, we use GitHub because we want to:
+
+ 1. easily collaborate on common files
+ 2. have advanced reviewing tools and version history
+ 3. keep a record of discussions, and every change made
+ 4. be able to revert back to previous versions, if needed
+ 5. render some of the static files in a web interface
```

Comment:
What do render, static files, and web interface mean?

Reply:
It's the previous point above! Let me know how I can change it to make it more clear.

Reply:
A little background on the [render terminology](https://www.freecodecamp.org/news/what-exactly-is-client-side-rendering-and-hows-it-different-from-server-side-rendering-bd5c786b340d/). Static files mean that they are just raw text, not a binary that needs to be executed, and web interface is just your browser (the little window you are looking at now!)

Reply:
I ~think~ I get it... Explaining each term with simple examples might be the move here for non-tech peeps. Raw text could mean markdown, or just 'writing', like on a Word doc. When you say binary all I know is that that's all 0s and 1s, but not what that means for a website. My ask would be for an explanation of all the terms using simple vocabulary and accessible examples.

Reply:
Updated.